index.html

```diff
@@ -25,15 +23,14 @@
 
       gtag("config", "G-L9EM4V5T72");
     </script>
->>>>>>> Application-deploy
```

Comment:
- [ ] There's a weird bar at the top of the page, kindly remove it. It is from a git merge.
![image](https://user-images.githubusercontent.com/49621279/122208937-cb688280-ce58-11eb-96c0-9c50c4d846ef.png)